frontends/android/goserver/goserver.go

```diff
@@ -209,3 +211,11 @@ func Serve(dataDir string, environment GoEnvironmentInterface, goAPI GoAPIInterf
 func Shutdown() {
 	bridgecommon.Shutdown()
 }
+
+func TriggerAuth() {
+	bridgecommon.TriggerAuth()
+}
+
+func AuthResult(ok bool) {
+	bridgecommon.AuthResult(ok)
+}
```

Comment:
docstrings missing (I know it is a bit pedantic :joy: but imo it is great for code hygiene to always document exported functions, even if they are trivial).

we need to fix that linter :P

Reply:
Eheh sorry, added